### PR TITLE
feat(graphical-selector): Differentiate node style for planned courses

### PR DESCRIPTION
**Pull request by leonardo-fan**
- config and utils for graphical selector
- determine whether course in planner or not an change colour based on it
- also was getting an error for linter script yesterday and today because it couldn't find ts and tsx scripts in the directory so removed them from the script

https://user-images.githubusercontent.com/90736577/180131777-371d2813-010d-4d55-a4b0-1231d8d61736.mov

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
     "build": "vite build",
     "preview": "vite preview --host",
     "test": "jest",
-    "lint": "eslint src/**/*.{js,jsx,ts,tsx}",
-    "lint:fix": "eslint --fix src/**/*.{js,jsx,ts,tsx}"
+    "lint": "eslint src/**/*.{js,jsx}",
```
Comment (gtangelo) [inline frontend/package.json:43]: Oooof 💀

File: frontend/src/components/PrerequisiteTree/config.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const defaultNode = {
   },
 };
 
-const prereqNodeAddtionalStyle = (courseName) => ({
+const prereqNodeAdditionalStyle = (courseName) => ({
```
Comment (gtangelo) [inline frontend/src/components/PrerequisiteTree/config.js:30]: did u manage to spell addtional this time? hehe

File: frontend/src/pages/GraphicalSelector/utils.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import GRAPH_STYLE from "./config";
+
+// plannedCourses is an object of with keys of courseCodes
+const handleNodeData = (courseCode, plannedCourses) => {
+  // determine if planned or unplanned
+  if (Object.prototype.hasOwnProperty.call(plannedCourses, courseCode)) {
```
Comment (gtangelo) [inline frontend/src/pages/GraphicalSelector/utils.js]: Nitpick: why dont u do `plannedCourses[courseCode]` instead? 👀  Either way works just curios lol

**Review by gtangelo — APPROVED** [review]
lgtm!

File: frontend/src/components/PrerequisiteTree/config.js (hunk 2 above)
Comment (leonardo-fan) [reply frontend/src/components/PrerequisiteTree/config.js:30]: BAHAHHAHA 😎

Comment (leonardo-fan) [reply frontend/src/components/PrerequisiteTree/config.js:30]: yep :)

File: frontend/src/pages/GraphicalSelector/utils.js (hunk 3 above)
Comment (leonardo-fan) [reply frontend/src/pages/GraphicalSelector/utils.js]: truee lul - tbh I used hasOwnProperty before but the linter didn't like it so I googled and they recommended this xd

**Pull request merged**